<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="8" passed="5" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-02T02:48:57 EEST" name="All Test Suite" finished-at="2024-08-02T02:52:06 EEST" duration-ms="189363">
    <groups>
    </groups>
    <test started-at="2024-08-02T02:48:57 EEST" name="E:/Rossy/Skillo/SKAU14TAF" finished-at="2024-08-02T02:52:06 EEST" duration-ms="189363">
      <class name="com.skilo.gui.RegistrationTest">
        <test-method is-config="true" signature="setUpTest()[pri:0, instance:com.skilo.gui.RegistrationTest@67c27493]" started-at="2024-08-02T02:51:47 EEST" name="setUpTest" finished-at="2024-08-02T02:51:48 EEST" duration-ms="1922" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyUserCanRegisterWithValidData()[pri:0, instance:com.skilo.gui.RegistrationTest@67c27493]" started-at="2024-08-02T02:51:48 EEST" name="verifyUserCanRegisterWithValidData" finished-at="2024-08-02T02:51:56 EEST" duration-ms="7815" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserCanRegisterWithValidData -->
        <test-method is-config="true" signature="tearDownTest(org.testng.ITestResult)[pri:0, instance:com.skilo.gui.RegistrationTest@67c27493]" started-at="2024-08-02T02:51:56 EEST" name="tearDownTest" finished-at="2024-08-02T02:51:57 EEST" duration-ms="410" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyUserCanRegisterWithValidData status=SUCCESS method=RegistrationTest.verifyUserCanRegisterWithValidData()[pri:0, instance:com.skilo.gui.RegistrationTest@67c27493] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method is-config="true" signature="setUpTest()[pri:0, instance:com.skilo.gui.RegistrationTest@67c27493]" started-at="2024-08-02T02:51:57 EEST" name="setUpTest" finished-at="2024-08-02T02:51:58 EEST" duration-ms="1205" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyUserCanNotRegisterWithINValidData()[pri:0, instance:com.skilo.gui.RegistrationTest@67c27493]" started-at="2024-08-02T02:51:58 EEST" name="verifyUserCanNotRegisterWithINValidData" finished-at="2024-08-02T02:52:05 EEST" duration-ms="6950" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at org.testng.Assert.assertFalse(Assert.java:89)
at com.skilo.gui.RegistrationTest.verifyUserCanNotRegisterWithINValidData(RegistrationTest.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserCanNotRegisterWithINValidData -->
        <test-method is-config="true" signature="tearDownTest(org.testng.ITestResult)[pri:0, instance:com.skilo.gui.RegistrationTest@67c27493]" started-at="2024-08-02T02:52:05 EEST" name="tearDownTest" finished-at="2024-08-02T02:52:06 EEST" duration-ms="1476" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyUserCanNotRegisterWithINValidData status=FAILURE method=RegistrationTest.verifyUserCanNotRegisterWithINValidData()[pri:0, instance:com.skilo.gui.RegistrationTest@67c27493] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
      </class> <!-- com.skilo.gui.RegistrationTest -->
      <class name="com.skilo.gui.LoginTest">
        <test-method is-config="true" signature="setupTestSuite()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:48:55 EEST" name="setupTestSuite" finished-at="2024-08-02T02:48:57 EEST" duration-ms="1895" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTestSuite -->
        <test-method is-config="true" signature="setUpTest()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:48:57 EEST" name="setUpTest" finished-at="2024-08-02T02:49:03 EEST" duration-ms="6010" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyAlreadyRegisteredUserCanSuccessfullyLogin()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:49:03 EEST" name="verifyAlreadyRegisteredUserCanSuccessfullyLogin" finished-at="2024-08-02T02:49:12 EEST" duration-ms="8817" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAlreadyRegisteredUserCanSuccessfullyLogin -->
        <test-method is-config="true" signature="tearDownTest(org.testng.ITestResult)[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:49:12 EEST" name="tearDownTest" finished-at="2024-08-02T02:49:12 EEST" duration-ms="625" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyAlreadyRegisteredUserCanSuccessfullyLogin status=SUCCESS method=LoginTest.verifyAlreadyRegisteredUserCanSuccessfullyLogin()[pri:0, instance:com.skilo.gui.LoginTest@49139829] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method is-config="true" signature="setUpTest()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:49:12 EEST" name="setUpTest" finished-at="2024-08-02T02:49:14 EEST" duration-ms="1775" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongPassword()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:49:14 EEST" name="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongPassword" finished-at="2024-08-02T02:50:04 EEST" duration-ms="49703" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongPassword -->
        <test-method is-config="true" signature="tearDownTest(org.testng.ITestResult)[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:50:04 EEST" name="tearDownTest" finished-at="2024-08-02T02:50:04 EEST" duration-ms="288" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongPassword status=SUCCESS method=LoginTest.verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongPassword()[pri:0, instance:com.skilo.gui.LoginTest@49139829] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method is-config="true" signature="setUpTest()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:50:04 EEST" name="setUpTest" finished-at="2024-08-02T02:50:06 EEST" duration-ms="1302" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongUsername()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:50:06 EEST" name="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongUsername" finished-at="2024-08-02T02:50:55 EEST" duration-ms="49268" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongUsername -->
        <test-method is-config="true" signature="tearDownTest(org.testng.ITestResult)[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:50:55 EEST" name="tearDownTest" finished-at="2024-08-02T02:50:55 EEST" duration-ms="317" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongUsername status=SUCCESS method=LoginTest.verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongUsername()[pri:0, instance:com.skilo.gui.LoginTest@49139829] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method is-config="true" signature="setUpTest()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:50:55 EEST" name="setUpTest" finished-at="2024-08-02T02:50:57 EEST" duration-ms="2089" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithNotProvidedCredentials()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:50:57 EEST" name="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithNotProvidedCredentials" finished-at="2024-08-02T02:51:46 EEST" duration-ms="48841" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithNotProvidedCredentials -->
        <test-method is-config="true" signature="tearDownTest(org.testng.ITestResult)[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:51:46 EEST" name="tearDownTest" finished-at="2024-08-02T02:51:46 EEST" duration-ms="448" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithNotProvidedCredentials status=SUCCESS method=LoginTest.verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithNotProvidedCredentials()[pri:0, instance:com.skilo.gui.LoginTest@49139829] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method is-config="true" signature="cleanFiles()[pri:0, instance:com.skilo.gui.LoginTest@49139829]" started-at="2024-08-02T02:52:06 EEST" name="cleanFiles" finished-at="2024-08-02T02:52:06 EEST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanFiles -->
      </class> <!-- com.skilo.gui.LoginTest -->
      <class name="com.skilo.gui.PostTest">
        <test-method signature="verifyUserCanCreateNewPost(java.io.File,java.lang.String)[pri:0, instance:com.skilo.gui.PostTest@77fbd92c]" started-at="2024-08-02T02:51:46 EEST" name="verifyUserCanCreateNewPost" data-provider="PostTestDataProvider" finished-at="2024-08-02T02:51:46 EEST" duration-ms="4" status="FAIL">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void com.skilo.gui.PostTest.verifyUserCanCreateNewPost(java.io.File,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserCanCreateNewPost([java.io.File arg0, java.lang.String arg1])
Arguments: [(java.lang.String) testingDemos,(java.lang.String) testing,(java.lang.String) testingDemos,(java.io.File) src\test\resources\uploads\testUpload.jpg,(java.lang.String) Testing create post caption]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.skilo.gui.PostTest.verifyUserCanCreateNewPost(java.io.File,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserCanCreateNewPost([java.io.File arg0, java.lang.String arg1])
Arguments: [(java.lang.String) testingDemos,(java.lang.String) testing,(java.lang.String) testingDemos,(java.io.File) src\test\resources\uploads\testUpload.jpg,(java.lang.String) Testing create post caption]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:915)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserCanCreateNewPost -->
        <test-method signature="verifyUserCanCreateNewPostAndDeleteIt(java.io.File,java.lang.String)[pri:0, instance:com.skilo.gui.PostTest@77fbd92c]" started-at="2024-08-02T02:51:47 EEST" name="verifyUserCanCreateNewPostAndDeleteIt" data-provider="PostTestDataProvider" finished-at="2024-08-02T02:51:47 EEST" duration-ms="1" status="FAIL">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void com.skilo.gui.PostTest.verifyUserCanCreateNewPostAndDeleteIt(java.io.File,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserCanCreateNewPostAndDeleteIt([java.io.File arg0, java.lang.String arg1])
Arguments: [(java.lang.String) testingDemos,(java.lang.String) testing,(java.lang.String) testingDemos,(java.io.File) src\test\resources\uploads\testUpload.jpg,(java.lang.String) Testing create post caption]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.skilo.gui.PostTest.verifyUserCanCreateNewPostAndDeleteIt(java.io.File,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserCanCreateNewPostAndDeleteIt([java.io.File arg0, java.lang.String arg1])
Arguments: [(java.lang.String) testingDemos,(java.lang.String) testing,(java.lang.String) testingDemos,(java.io.File) src\test\resources\uploads\testUpload.jpg,(java.lang.String) Testing create post caption]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:915)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserCanCreateNewPostAndDeleteIt -->
      </class> <!-- com.skilo.gui.PostTest -->
    </test> <!-- E:/Rossy/Skillo/SKAU14TAF -->
  </suite> <!-- All Test Suite -->
</testng-results>
